x-pack/plugins/apm/public/components/app/Main/route_config/route_handlers/agent_configuration.tsx

```diff
@@ -40,6 +41,7 @@ export function EditAgentConfigurationRouteHandler() {
 }
 
 export function CreateAgentConfigurationRouteHandler() {
+  const history = useHistory();
```

Comment:
Is this the same `history` object as the platform `history`? If not, is that ok?

Reply:
Yes. `useHistory` uses the `history` provided to the router, which we do here: https://github.com/elastic/kibana/pull/74328/files#diff-0d2b10d4756fabbe9a3043f569b0b9d3R93.

Reply:
Perfect!